### PR TITLE
Update reset-the-audio-conferencing-pin-in-teams.md

### PR DESCRIPTION
**Pull request by ruizoscar**
Removing a deprecated behavior

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/reset-the-audio-conferencing-pin-in-teams.md [hunk 1]
```diff
@@ -55,10 +55,8 @@ Meetings can be started when an authenticated user joins using the Microsoft Tea
 
 - For security purposes, the PIN is only shown to an administrator on one time, when the PIN is reset. After the PIN is reset by an administrator, the PIN will be listed as ***********.
```
Comment (JohanFreelancer9) [inline Teams/reset-the-audio-conferencing-pin-in-teams.md:56]:
```suggestion
- For security purposes, the PIN is only shown to an administrator on one occasion, when the PIN is reset. After the PIN is reset by an administrator, the PIN will be listed as ***********.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@ruizoscar - please also commit my minor additional edit. Thanks.

**Comment by scanum** [comment]
@ruizoscar Following up here.

**Pull request closed without merging**